.env

```diff
@@ -0,0 +1 @@
+EBSI_BEARER="eyJhbGciOiJFUzI1NksiLCJ0eXAiOiJKV1QiLCJraWQiOiJkaWQ6ZWJzaTp6cjJyV0RISHJVQ2RaQVc3d3NTYjVuUSNrZXlzLTEifQ.eyJvbmJvYXJkaW5nIjoicmVjYXB0Y2hhIiwidmFsaWRhdGVkSW5mbyI6eyJzdWNjZXNzIjp0cnVlLCJjaGFsbGVuZ2VfdHMiOiIyMDIzLTAzLTE2VDE0OjMzOjI4WiIsImhvc3RuYW1lIjoiYXBwLXBpbG90LmVic2kuZXUiLCJzY29yZSI6MC45LCJhY3Rpb24iOiJsb2dpbiJ9LCJpc3MiOiJkaWQ6ZWJzaTp6cjJyV0RISHJVQ2RaQVc3d3NTYjVuUSIsImlhdCI6MTY3ODk3OTUyMSwiZXhwIjoxNjc4OTgwNDIxfQ.eY0dZSrkRrq5Wmg5IAsP_mKF_1fQu4NToqHQuIg0sB75TBkzkqlx0r_ucZFZSkWas2C5DNRgImFn6-4Fk5G42w"
```

Comment:
Hmm, we can also remove this file and pass bearer token as CMD arg: `BEARER=ey... ebsi_test.ts`